src/lib/filters-aside.svelte

```diff
@@ -1,6 +1,33 @@
+<script>
+    import FilterButtonAuthor from '$lib/filter-button-author.svelte';
+
+    let { author = $bindable(), authors, totalAuthors, authorsPage } = $props();
+    function clearAuthor() {
+        author = null;
+    }
+</script>
+<div class="big-screen-only">
 
 <aside>
-    <span>Filters</span>
+    <p class="summary">Filters</p>
+    {#if author}
+        <noscript>
+            <p class="current-filter">{author}</p>
+        </noscript>
+        <button class="js-only badge btn-primary" onclick={clearAuthor} aria-label="remove author filter">                                
+            <svg
```

Comment:
Perhaps it would be cleaner to also make this one a .svg file and put it into the static folder